study-builder/studies/testboston/emails/kit_arrived.en.html

```diff
@@ -1,14 +1,16 @@
 #set( $text = {
-  "subject": "Your test kit was delivered!",
+  "subject": "Your kit will be delivered very soon",
```

Comment:
The subject should also be updated in the `study-emails.conf` configuration file.